models/users.js

```diff
@@ -372,7 +376,13 @@ const addForVerification = async (userId, discordId, profileImageUrl, discordIma
     userId,
     discordId,
     discord: { url: discordImageUrl, approved: false, date: admin.firestore.Timestamp.fromDate(new Date()) },
-    profile: { url: profileImageUrl, approved: false, date: admin.firestore.Timestamp.fromDate(new Date()) },
+    profile: {
+      url: profileImageUrl,
+      publicId: profileImageID,
+      approved: false,
+      date: admin.firestore.Timestamp.fromDate(new Date()),
```

Comment:
what is date here?
Did you mean createdAt?

Reply:
again, this was already present there, therefore I opted for using the current ones only. But I can make the change to createdAt, as that will make more sense.

Reply:
Or, I'll rather use `updatedAt` as the one single picture entry in the photo-verification object can be updated.